web-frontend/src/pages/TransferCoins/SendCoins/SendForm.tsx

```diff
@@ -37,7 +39,10 @@ export function SendForm({ ...props }) {
   );
   const [fees, setFees] = useState(data?.fees ?? '1000');
   const [recipient, setRecipient] = useState(data?.recipient ?? '');
```

Comment:
```suggestion
  const [fees, setFees] = useState<string>(data?.fees ?? '1000');
  const [recipient, setRecipient] = useState<string>(data?.recipient ?? '');
```

Reply:
Yes I saw that and I figured I would do a pr for it but I guess I can do it here

Reply:
> Yes I saw that and I figured I would do a pr for it but I guess I can do it here

Ye! You can do it here if you want because it's just 2 line 😉